src/routes/groups/groups.router.js

```diff
@@ -0,0 +1,87 @@
+const express = require('express');
+const { listGroup, postGroup, updateGroup, deleteGroup } = require("./groups.controller");
```

Comment:
How might you redesign the groups controller module and imports so that the method names are just the action (e.g. list, post, etc), *without* losing the context of the Group?